autopush/web/registration.py

```diff
@@ -133,6 +137,41 @@ def validate_data(self, data):
                                          errno=109,
                                          headers=request_pref_header)
 
+    @post_load
+    def handler_kwargs(self, data):
+        # not used
```

Comment:
this just pops it off the handler kwargs after it's validated (not used = not used by the handlers)